setup.py

```diff
@@ -139,3 +135,7 @@ def read(*names, **kwargs):
     compile()
 except:
     pass
+
+if os.path.exists(".git/hooks"):  # check if we are in git repo
```

Comment:
Is there a reason for moving this to the bottom?

Reply:
No, I don't know how. I would correct it.

Reply:
I think you can try `git checkout HEAD -- setup.py` to reset it to head.